src/common/LogClient.h

```diff
@@ -39,6 +39,15 @@ namespace logging {
 }
 }
 
+#if defined(WITH_SEASTAR) && !defined(WITH_ALIEN)
+namespace crimson::net {
+  class Messenger;
+}
+#define messenger_type crimson::net::Messenger
```

Comment:
messanger_type does not conform to the style-guide naming for types. I know there is a macro involved,
still - I'd prefer
using messenger_t = crimson::net::Messenger etc'

Reply:
i.e. - the main point is 'messenger_t'; a secondary issue - maybe use 'using' instead of a macro